cumulus/node/README.md

```diff
@@ -57,22 +61,20 @@ Now the primary node should be producing blocks.
 Ensure the bootnode for the primary node is correct and run this command to start an executor:
 
 ```bash
-$ ./target/release/subspace-executor \
-    --alice \
-    --collator \
-    --force-authoring \
-    --base-path first-db \
-    --port 40333 \
-    --log=cirrus=trace,txpool=trace,gossip=trace \
-    --rpc-port 8845 \
-    --ws-port 8846 \
+$ ./target/release/subspace-node \
+    --chain dev \
+    -d db1 \
+    --log=trace \
+    --bootnodes "/ip4/127.0.0.1/tcp/30333/p2p/PRIMARY_PEER_ID" \
+    --port 30343 \
+    --ws-port 9977 \
     -- \
-        --validator \
-        --log=trace \
-        --chain dev.json \
-        --bootnodes "/ip4/127.0.0.1/tcp/30333/p2p/PRIMARY_PEER_ID" \
-        --port 30343 \
-        --ws-port 9977
+        --alice \
```

Comment:
Why no `--collator` and `--force-authoring` this now?

Reply:
Actually, they are unnecessary before this PR.

Reply:
`--collator` was an alias for `--validator` with a different name (for convenience I guess, should really be renamed in Substrate itself if you ask me to something more generic).